src/store/reducers.js

```diff
@@ -26,6 +27,79 @@ function computeRebootStats(remediation) {
   };
 }
 
+export const remediationSystems = ({ LOAD_ENTITIES_FULFILLED }) =>
+  applyReducerHash({
+    [ACTION_TYPES.SELECT_ENTITY]: (state, { payload }) => {
+      const selected = state.selected || new Map();
```

Comment:
Maps are mutable and can cause issues within redux. Can we use a plain object instead?

Reply:
The map is specifically used so we don't have to run `filters` and `finds` and removing the columns. I'll mark this as `TODO: refactor` and we can fix it later on.